backend/pom.xml

```diff
@@ -47,6 +52,11 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.security</groupId>
+			<artifactId>spring-security-oauth2-jose</artifactId>
+			<version>5.5.2</version>
```

Comment:
This overrides the manage version, why is this required?

Reply:
Let my IDE import, didn't notice it pulled a specific version. 
Fixed